Tests/FirebaseTests/Models/DynamicLink+Dummy.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+@testable import Firebase
+
+extension DynamicLink {
+    
+    static var dummy: DynamicLink {
+        return DynamicLink(shortLink: URL(string: "https://shortLink")!, previewLink: URL(string: "https://previewLink")!, warning: nil)
+    }
```

Comment:
you can simplify this property definition like so:

```swift
static let dummy = DynamicLink(shortLink: URL(string: "https://shortLink")!, previewLink: URL(string: "https://previewLink")!, warning: nil)
```

since it is a static, you can declare it in an extension. This is only stored properties (aka: mutable instance properties) that are not allowed to be declared in an extension.

Reply:
41ba0cf